frontend/device/android/device.lua

```diff
@@ -230,22 +230,21 @@ end
 
 function Device:initNetworkManager(NetworkMgr)
     function NetworkMgr:turnOnWifi(complete_callback)
```

Comment:
It seems that if the `complete_callback` thing is useless here you could just as well write `NetworkMgr.turnOnWifi = android.openWifiSettings`? But whichever, writing it this way is probably friendlier to potential future changes.

Reply:
yup, ignoring is a valid way of treating a parameter :smile: